### PR TITLE
Created header and showcase section.

### PR DESCRIPTION
**Pull request by JakariaJishan**
### Project requirements:

- For this project, I use the [HTML&CSS linter]

_The 2 sections  I built for this project are:_

1. the toolbar (or header):
2.Toolbar

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi @JakariaJishan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that does not have a professional README. A good README will provide summary information about the project for people visiting your repository. Kindly update your README file using the [template](https://github.com/microverseinc/readme-template) provided by Microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,156 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [github-flow] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```
Comment (nedjwab) [inline README.md]: [ ] customize the README file with your own pieces of information: project description, author name, stack used ... kindly fix that 🤓

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<body>
+    <nav>
+        <header>
+            <a href="#">
+                <img class="brang-img" src="./images/Brand-logo.png" alt="brang-logo" />
+            </a>
+            <img class="hamburger-menu" src="./images/Icon - Menu.png" alt="hamburger-menu">
+        </header>
+    </nav>
+    <div class="showcase">
```
Comment (nedjwab) [inline index.html]: [ ] The container of the section is `a section tag`.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. kindly fix that 🤓

File: index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <div class="hero">
+            <h1>Hey There.
+                I’m Jakaria</h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="icons">
```
Comment (nedjwab) [inline index.html:30]: [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social net works with the icons, using href. kindly fix that 🤓

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @JakariaJishan  

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
 
check the comments below 👇👇
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Gopxfs — APPROVED** [review]
# 🟢 STATUS: APPROVED 🟢

## Hello @JakariaJishan 👋,

Your project is complete! You followed all the requirements, feel free to merge it! 😄
Congratulations! 🎉 



## Highlights
✔️ Linters are passing successfully.
✔️ Implemented all the changes requested by the previous reviewer. Well done!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**